lib/evm_watchdog.rb

```diff
@@ -78,7 +78,7 @@ def self.start_evm
 
   def self.log_info(message)
     File.open('/var/www/miq/vmdb/log/evm.log', 'a') do |f|
-      f.puts "[----] I, [#{Time.now.utc.iso8601(6)} ##{Process.pid}:#{Thread.current.object_id.to_s(16)}]  INFO -- : EvmWatchdog - #{message}"
+      f.puts "[----] I, [#{Time.now.localtime.strftime("%Y-%m-%dT%H:%M:%S.%6N".freeze)} ##{Process.pid}:#{Thread.current.object_id.to_s(16)}]  INFO -- : EvmWatchdog - #{message}"
```

Comment:
@jrafanie I think I can just use `Time.now`. That's what it looks like the built-in logger is doing here https://github.com/ruby/ruby/blob/trunk/lib/logger.rb#L472. What do you think?

Reply:
I agree, @gtanzillo.  I think we should really follow how ruby does it.

Reply:
@jrafanie I made the change to just do `Time.now` this should be ready to go.